modules/button/react/README.md

```diff
@@ -184,15 +184,16 @@ import {activityStreamIcon} from '@workday/canvas-system-icons-web';
 
 Default: `IconButtonVariant.Circle`
 
-| Theme           | Description                                      |
-| --------------- | ------------------------------------------------ |
-| `Square`        | Square, white background, dark gray icon         |
-| `SquareFilled`  | Square, gray background, dark gray icon          |
-| `Plain`         | Dark gray icon                                   |
-| `Circle`        | Circular, dark gray icon                         |
-| `CircleFilled`  | Circular, gray background, dark gray icon        |
-| `Inverse`       | Circular, transparent background, white icon     |
-| `InverseFilled` | Circular, semitransparent background, white icon |
+| Theme           | Description                                                |
+| --------------- | ---------------------------------------------------------- |
+| `Alternate`     | Square, white background, dark gray icon used for toolbars |
```

Comment:
I made this specific to toolbars because it has specific styles for that reason, thoughts on the wording? @jpante